### PR TITLE
[ISSUE #2138]for ShenyuClientRegisterDubboServiceImpl test

### PR DESCRIPTION
**Pull request by sunrui849**
test for  ShenyuClientRegisterDubboServiceImpl #2138

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.springframework.context.ApplicationEventPublisher;
+
+import java.util.*;
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java:43]: donnot used  import *

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+/**
+ * Test cases for ShenyuClientRegisterDubboServiceImpl.
+ */
+@PrepareForTest(MetaDataServiceImpl.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
+public final class ShenyuClientRegisterDubboServiceImplTest {
+
+    @Mock
+    private MetaDataMapper metaDataMapper;
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java:57]: keep line

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                .pathDesc("pathDescMetaData")
+                .rpcType("rpcTypeMetaData")
+                .serviceName("serviceName3")
+                .methodName("methodName3")
+                .parameterTypes("parameterTypesMetaData")
+                .rpcExt("rpcExtMetaData")
+                .enabled(true)
+                .build();
+    }
+
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java:167]: remove Empty line

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java (hunk 1 above)
Comment (sunrui849) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java]: done

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java (hunk 2 above)
Comment (sunrui849) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java:62]: done

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java (hunk 3 above)
Comment (sunrui849) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterDubboServiceImplTest.java]: done

**Comment by yu199195** [comment]
pls check ci~

**Comment by sunrui849** [comment]
> pls check ci~

**Pull request closed without merging**